infra/resources/prod/data.tf

```diff
@@ -103,3 +107,8 @@ data "azurerm_storage_account" "iopstexportdata" {
   name                = "iopstexportdata"
   resource_group_name = data.azurerm_resource_group.weu_operations.name
 }
+
+data "azurerm_monitor_action_group" "error_action_group" {
+  resource_group_name = data.azurerm_resource_group.weu_common.name
+  name                = replace("${local.project_legacy}error", "-", "")
+}
```

Comment:
Currently we are moving alerts related to our domain into another action group as you can see [here](https://github.com/pagopa/io-infra/pull/1164), I think we can use this new action group here